cmd/event-logger/main.go

```diff
@@ -167,5 +170,38 @@ func main() {
 		fmt.Fprint(w, "</tbody></table></body>")
 	})
 
+	http.HandleFunc("/emit/{detailType}", func(w http.ResponseWriter, r *http.Request) {
+		if r.Method == http.MethodPost {
+			sqsClient := &sqsClient{
+				svc:      sqs.NewFromConfig(cfg),
+				queueURL: "http://localhost:4566/000000000000/event-bus-queue",
+			}
+
+			detailType := r.PathValue("detailType")
+
+			var detail json.RawMessage
+			if err := json.NewDecoder(r.Body).Decode(&detail); err != nil {
+				log.Printf("failed to unmarshal %s: %v", detailType, err)
+			}
+
+			var sendErr error
+
+			switch detailType {
```

Comment:
I'd remove this so it always sends the message, I basically don't want to ever have to edit this file. We should treat `event-logger` as something that could be extracted from this repo and be useful for anyone